src/styles/_header.scss

```diff
@@ -37,5 +36,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: 300ms ease;
```

Comment:
you need to specify the transition property just like you did in `.catalog__card` 😉